BUILDING.md

```diff
@@ -53,23 +54,34 @@ $ git lfs checkout
 
 ### Gradle
 
-After cloning you can build LP with `./gradlew build` and find the output in
+After cloning, you can build LP with `./gradlew build` and find the output in
 `build/libs`. If the task fails there may be something wrong with maven
 repositories or [a Java update broke ForgeGradle 2](https://github.com/MinecraftForge/ForgeGradle/issues/652)
 or something may be wrong with your setup. You may definitely ask for help on
 the mentioned communication channels above, but please be sure to state your
 issue as good as possible and be nice to others.
 
+### IDEA Quirks
+
+If you opened the project in IDEA before running the `./gradlew build` command the run
+configurations will not be visible, simple fix is to reload the project.
+
 
 ### Running Minecraft from your dev environment
 
 After you successfully built LP you can probably run Minecraft directly from
 your IDE. There may be a ton of missing texture errors and missing language
 files though and this problem even happens when using a new clean mod template,
 but don't be sad! The solution is to copy your `build/resources/main/*` into
-`build/classes/java/main` before launching the game or you may link those two
- together:
+`build/classes/java/main` before launching the game, or you may link those two
+ together (Linux or WSL):
```

Comment:
Did you test this on WSL?

Reply:
Yes, that is how I initially set it up, but then I tried rewriting it in cmd just for fun ;)